### ISSUE TITLE
wrong field length on csv import that creates table

### ISSUE THREAD

**Issue by msanchez78** [phpmyadmin/phpmyadmin#13292]
### Steps to reproduce
1. import any csv file where the field lengths in the first few records are smaller than subsequent records
2. to a new table - so it creates a new table
3. example: the 3rd field is Account_Name. the first few records have names like Jane Mary Joe AJ - largest is char(4). but the remaining records have long names like Michael Matthew. the import fails because the table was created with this field data type/sizs VARCHAR(4)

### Expected behaviour
the field sizes for the new table have to be large enough to import all fields in all records. in the above example, field Account_Name s/b VARCHAR(8)

### Actual behaviour
get error code #1406 - Data too long for column 'Account_Name' at row 1 

### Server configuration
**Operating system**: Linux

**Web server:**
AWS
**Database:**
MySQL
**PHP version:**

**phpMyAdmin version:**

### Client configuration
**Browser:**
ALL:
Apache/2.4.6 (Red Hat Enterprise Linux) PHP/5.4.16
Database client version: libmysql - mysqlnd 5.0.10 - 20111026 - $Id: e707c415db32080b3752b232487a435ee0372157 $
PHP extension: mysqli Documentation
PHP version: 5.4.16

**Comment by aycanadalwork** [comment]
Got the same problem.

**Comment by ibennetch** [comment]
Possibly related: https://stackoverflow.com/questions/47543250/configure-columns-when-importing-csv-into-mysql

**Comment by williamdes** [comment]
@msanchez78 The request is to check length before import, right ?

**Comment by aycanadalwork** [comment]
Yes sir check length on all rows to determine field max length.

**Comment by msanchez78** [comment]
Yes check the max lenght of each field which means you have to parse every
row this way you can correctly create the data type and related size. Thank
you.
